packages/hash/frontend/src/components/BlockLoader/BlockLoader.tsx

```diff
@@ -80,6 +79,9 @@ export const BlockLoader: FunctionComponent<BlockLoaderProps> = ({
   onBlockLoaded,
   // shouldSandbox,
 }) => {
+  const { authenticatedUser } = useAuthenticatedUser();
+  const accountId = authenticatedUser?.userAccountId ?? "";
```

Comment:
Using `undefined` is a lot less misleading than using `""`.
```suggestion
  const accountId = authenticatedUser?.userAccountId;
```

If using `undefined` introduces to many type errors, than maybe it's worth leaving a `@todo` here.

Reply:
I changed it up in to get rid of the empty string 1e8b55bdf2abf0decbb6b59f45adf6c2d45efee3